src/components/groupedcards.js

```diff
@@ -22,7 +22,7 @@ define(['dom', 'appRouter', 'connectionManager'], function (dom, appRouter, conn
                     return void appRouter.showItem(items[0]);
                 }
 
-                var url = 'itemdetails.html?id=' + itemId + '&serverId=' + serverId;
+                var url = 'details?id=' + itemId + '&serverId=' + serverId;
```

Comment:
Why was this changed?

Reply:
The page was moved to the controller, so the URL would be `/controller/itemDetails/itemldetails.html`.

To make it cleaner, I added aliases to the router, so that the routes can be overriden as we cleanup the HTML files in the root folder.